lib/fog/vsphere/requests/compute/list_child_snapshots.rb

```diff
@@ -0,0 +1,68 @@
+module Fog
+  module Compute
+    class Vsphere
+      class Real
+        def list_child_snapshots(snapshot, opts = {})
+          s = Hash === snapshot ?
+            Snapshot.new(snapshot.update(service: self)) : snapshot
+          s.tree_node.childSnapshotList.map do |snap|
```

Comment:
Sorry, it is not possible because it is a method of Vsphere API provided by rbvmomi library.